assets/stylesheet/index.css

```diff
@@ -0,0 +1,208 @@
+header:first-child {
+  background-color: red;
+  font-size: 2vh;
+  position: sticky;
+  z-index: 1;
+  top: 0;
+}
+
+.auth {
+  font-size: 1.5vh;
+  height: auto;
+}
+
+nav ul {
+  display: flex;
+  list-style: none;
+}
+
+nav ul li {
+  flex-grow: 1;
+  border-right: 1px solid gray;
+  text-align: center;
+}
+
+nav ul a {
+  font-size: 1rem;
+  font-weight: bold;
+  color: #000;
+}
+
+footer ul li {
+  line-height: 0.1;
+  font-size: 12px;
+  border-right: 1px solid #dee2e6;
+}
+
+nav ul li:last-child {
+  border-right: none;
+}
+
+.option a {
+  color: red;
+}
+
+header a:hover {
+  text-decoration: none;
+  border-bottom: 2px solid red;
+}
+
+nav ul a:hover {
+  color: #000;
+}
+
+input {
+  width: 4rem;
+  border: none;
+  font-size: 1rem;
+}
+
+i {
+  color: gray;
+}
+
+.option {
+  background-color: #eae8e9;
+  padding: 5px 10px;
+  position: absolute;
+  bottom: 0;
+  left: 30;
+}
+
+.fixed-img {
+  width: 30vh;
+  height: auto;
+  fill: #fff;
+}
+
+.navbar::before {
+  position: absolute;
+  top: 0;
+  right: 0;
+  content: "";
+  display: block;
+  width: 3%;
+  background-color: #f72210;
+  border-left: 0.75rem solid white;
+  border-top: 0.8125rem solid red;
+}
+
+.opinionssection a:hover {
+  color: red !important;
+  text-decoration: none !important;
+  border-bottom: none;
+}
+
+.border-solidred {
+  border-top: 5px solid red !important;
+}
+
+.solidblue {
+  border-top: 5px solid #1c87fb !important;
+}
+
+.tube {
+  display: block;
+  background-color: #d9d9d9;
+  height: 5px;
+  border: 1px solid gray;
+}
+
+.fontsized {
+  font-size: 1.5rem !important;
```

Comment:
Kindly remove all `important` tags here and also all other `!important` tags. To override bootstrap styling consider chaining CSS selectors to increase specificity.

Reply:
I removed most of the `important` however i tried chaining CSS in two of them it was impossible since bootstrap also uses `!important `, it was overriding my CSS props, so i decided to leave **2** and I deleted **8** also modified the size of the buttons on small size.